### ISSUE TITLE
[3.0] Primary button colour contrast is not high enough

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/cwp-starter-theme#118]
**Starter 3.0.x-dev**

Example:

![image](https://user-images.githubusercontent.com/5170590/56938164-7bcd7c80-6b54-11e9-93e8-3a48c5fdf76f.png)

Axe report:

>Element has insufficient color contrast of 3.97 (foreground color: #ffffff, background color: #007bff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

cc @silverstripeux

**Comment by robbieaverill** [comment]
This is an issue with Bootstrap 4 - see https://github.com/twbs/bootstrap/issues/25126

@clarkepaul suggested using the old theme colours for the parts that don't meet the contrast requirements.
